test/suite/test_txn16.py

```diff
@@ -95,7 +95,7 @@ def run_toggle(self, homedir):
                 # There can be overlap with the log files that were
                 # there originally. Because some pages are rolled back
```

Comment:
Can you revert this comment also to the original one before it is changed in one of RTS commits?

Reply:
Fixed